apps/src/code-studio/components/SoundPicker.jsx

```diff
@@ -40,6 +40,7 @@ export default class SoundPicker extends React.Component {
     uploadsEnabled: PropTypes.bool.isRequired,
     showUnderageWarning: PropTypes.bool.isRequired,
     useFilesApi: PropTypes.bool.isRequired,
+    libraryOnly: PropTypes.bool.isRequired,
```

Comment:
Because this is marked as required, will there be errors when `showAssetManager` is called without specifying `libraryOnly`?